beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -3340,6 +3356,17 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         self.remove_notified(&block_root, r)
     }
 
+    /// Check for known and configured invalid block roots before processing.
+    pub fn check_invalid_block_roots(&self, block_root: Hash256) -> Result<(), BlockError> {
+        let is_invalid_holesky_block =
+            block_root == *INVALID_HOLESKY_BLOCK_ROOT && self.spec.deposit_chain_id == 17000;
```

Comment:
You should move this hardcoded value to `beacon_node/src/config.rs` and make it part of the flag parsing logic. Do we want to offer a way to run the binary and accept this invalid block root? i.e. make the default flag value equal to `INVALID_HOLESKY_BLOCK_ROOT` and if someone runs `--invalid-block-roots` the check is disabled. This can be useful to do Holesky testing or other purposes

Reply:
yeah I found myself wanting this the other day, I'll amend

Reply:
Addressed in https://github.com/sigp/lighthouse/pull/7042/commits/fe41e182199480ebba85e21aebfd31fa21dcc3d7